python/hail/expr/expressions/base_expression.py

```diff
@@ -272,8 +272,8 @@ def unify_all(*exprs) -> Tuple[Indices, LinkedList, LinkedList]:
         from collections import defaultdict
         sources = defaultdict(lambda: [])
         for e in exprs:
-            from .utils import get_refs
-            for name, inds in get_refs(e, *e._aggregations.exprs):
+            from .expression_utils import get_refs
+            for name, inds in get_refs(e, *[e for a in e._aggregations for e in a._exprs]):
```

Comment:
@tpoterba Can you check out this change?  I'm not familiar enough with the code.